src/rust/devcontainer-feature.json

```diff
@@ -36,14 +36,7 @@
                 "rust-lang.rust-analyzer",
                 "tamasfe.even-better-toml",
                 "serayuzgur.crates"
-            ],
-            "settings": {
-                "lldb.executable": "/usr/bin/lldb",
-                "files.watcherExclude": {
```

Comment:
Would it make sense to keep the watcherExclude? The target folder is the Rust compiler's output folder and I think also caches dependencies.